db/migrate/20201204160356_add_on_hold_to_vehicles.rb

```diff
@@ -0,0 +1,5 @@
+class AddOnHoldToVehicles < ActiveRecord::Migration[5.1]
+  def change
+    add_column :vehicles, :on_hold, :boolean, default: false
```

Comment:
https://edgeguides.rubyonrails.org/active_record_migrations.html#creating-a-table

We should consider setting null to false here to avoid: https://thoughtbot.com/blog/avoid-the-threestate-boolean-problem